### ISSUE TITLE
Cannot remove list entries in the "Missing Unity Events" panel.

### ISSUE THREAD

**Issue by VladimirMarko** [handzlikchris/Unity.MissingUnityEvents#11]
In this package's UI, in the "Missing Unity Events" panel, there's an editable list of types and their properties.
There is a button labeled "Add New Event" that adds a new entry to this list. There is no way to remove entries from this list - it can only grow. The entries persist over Unity restarts.

<img width="542" alt="cannot remove empty list entries" src="https://user-images.githubusercontent.com/22622584/156491846-9b40f844-6700-4133-947d-3662d7c42325.png">

In other words, I cannot get rid of the empty entries marked by the exclamation mark, only add more.

**EDIT**:
I figured out that you can reset everything via `Tools` -> `Missing Unity Events` -> `Recreate Config`. That's however not ideal, of course.

**Comment by handzlikchris** [comment]
Hi @VladimirMarko - thanks, glad you've found a workaround.

There shou'd be a '-' in that table. 

If not you can have a look with debugger what's happening, the code that draws it is in MissingUnityEventsManagerEditorWindow :172
```
 if (GUILayout.Button("-", PercentageWidth(8)))
{
    _config.EventConfigurationEntries.RemoveAt(i);
}
```

**Comment by VladimirMarko** [comment]
OK, I've been playing with this and I found out that consistently `PercentageWidth(66)` instead of `PercentageWidth(100)` means the whole window width for both `GUILayout` and `EditorGUILayout` static methods/UI elements.

I had no idea why. Then I checked DPI scaling and sure enough, it was 150%. So that's the problem. The way you use width is not robust with respect to DPI scaling (at least on Windows 10).

**Comment by VladimirMarko** [comment]
DPI scaling = 100%:

![dpi scaling 100%](https://user-images.githubusercontent.com/22622584/156623342-f545ea6a-ecc9-45f9-a2c8-c5aadcfcd55f.png)

DPI scaling = 150%:

<img width="960" alt="dpi scaling 150%" src="https://user-images.githubusercontent.com/22622584/156623349-c8d0d96a-13c7-48b2-9114-4ae477feb72e.png">

DPI scaling = 175%:

![dpi scaling 175%](https://user-images.githubusercontent.com/22622584/156623981-dfae2219-f0ee-4fa1-9283-c5ceaa0fb439.png)

**Comment by VladimirMarko** [comment]
Hm... I cannot figure out a clean solution to this.

The crux of the matter is that `GUILayout.Button("A window-wide button", GUILayout.Width(Screen.width));` produces a button that will be wider than its window, if DPI scaling is above 100%.

We have `Screen.dpi`, but that's the absolute value and not a scaling factor. The base value is system-dependent (e.g., normally 96 PPI on Windows) and the Unity API does not seem to allow us to get it for the current system.

Some external packages might help, but as I understand it, Unity does not play well with the NuGet package manager, so adding them as dependencies would be non-kosher, I suspect. I am not sure about this point.

In any case, I filed a bug report with Unity, as I assume that this discrepancy in behavior between `GUILayout.Width` and `Screen.width` is unintended.

As a workaround, I would suggest defining the UI elements using just absolute pixel values. I don't know whether there is a better way.

**Comment by VladimirMarko** [comment]
Nevermind. I figured it out.

Apparently the correct thing to use is `position.width` (inherited from `EditorWindow`), instead of `Screen.width`.

I will submit a pull request.

**Comment by handzlikchris** [comment]
Thanks @VladimirMarko that's some good investigation and fix
